src/views/ReferralProgram.vue

```diff
@@ -107,7 +109,7 @@
 </template>
```

Comment:
https://github.com/sora-xor/polkaswap-exchange-web/blob/f58e2b6b44eb171db8cd8e7ef267936dae4c9a4a/src/views/ReferralProgram.vue#L1

Here is should be `loading` property (data attribute from LoadingMixin), because the `loading `prop does not descend from the parent component